os/include/sys/syscall_lookup.h

```diff
@@ -0,0 +1,406 @@
+/****************************************************************************
```

Comment:
Is this file needed? We have syscall.csv which is used to generate similar mappings.

Reply:
Looks like we already have the file,
https://github.com/Samsung/TizenRT/blob/master/os/syscall/syscall_lookup.h